src/factories/interfaces/IPIM_WorkflowFactory_v1.sol

```diff
@@ -21,6 +21,10 @@ import {ERC20Issuance_v1} from "src/external/token/ERC20Issuance_v1.sol";
 import {IERC20} from "@oz/token/ERC20/IERC20.sol";
 
 interface IPIM_WorkflowFactory_v1 {
+    //--------------------------------------------------------------------------
+    // Errors
+    error PIM_WorkflowFactory__OnlyPimFeeRecipient();
```

Comment:
I believe we should add a `@notice` tag to errors as well, at least that is what we did so far (I believe, just checked a few contracts)?